src/app/components/Home/Office/Canvas/index.tsx

```diff
@@ -25,26 +29,30 @@ const useStyles = makeStyles({
   },
 
   placeField: {
-    opacity: "0",
     cursor: "pointer",
+    fill: "#c4c4c4",
     "&:hover": {
-      fill: "#299cff",
-      opacity: "1",
+      fill: "#299eff",
     },
   },
   placeLabel: {
     fontSize: "14px",
     lineHeight: "14px",
-    fill: "#fff",
+    fontWeight: "bold",
+    fill: "#000000",
     cursor: "pointer",
   },
 });
 
-const Canvas: React.FC = () => {  
+const setStrokeColor = (item: string, user: string) =>
+  item === "" && item !== user ? "green" : item === user ? "orange" : "red";
```

Comment:
better to avoid such multiple ternaries. You can try switch/case here:
```
switch (item) {
  case user:
    return "orange";
  case "":
    return "green";
  default:
    return "red"
}
```